app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.kt

```diff
@@ -170,6 +171,11 @@ class MoreBottomSheetFragment : BottomSheetDialogFragment() {
     fun uploadFeedback(feedback: Feedback) {
         val feedbackContentCreator = FeedbackContentCreator(requireContext(), feedback)
 
+        if (!isNetworkAvailable(requireContext())) {
+            Toast.makeText(requireContext(), R.string.error_feedback, Toast.LENGTH_LONG).show()
```

Comment:
Instead of toast, try having Snackbar with an action label (retry) to perform retry.

Reply:
So I should make show a snack bar which would have the option to reshow the Feedback Dialog. Am I right ?

Reply:
![image](https://github.com/user-attachments/assets/50469815-8a3a-4215-91b9-444ec15b997a)
Suggestion : Allow the user to perform retry if the feedback is not submitted. Wdyt?

Reply:
Allowing the user to retry would be great indeed.
Maybe leave the form there until sent successfully?